server/views/applications/pages/basic-information/reason-for-short-notice.njk

```diff
@@ -23,7 +25,7 @@
         fieldset: {
           legend: {
             text: page.question,
-            classes: "govuk-fieldset__legend--m"
+            classes: "govuk-fieldset__legend--{{ 'm' if page.title else 'l' }}"
```

Comment:
This needs to have the `isPageHeading` property set to `true` when there isn't a title https://design-system.service.gov.uk/components/radios/